java/selenium/manual-mode/src/test/java/com/deque/watcher_examples/selenium/ManualModeTest.java

```diff
@@ -0,0 +1,103 @@
+package com.deque.watcher_examples.selenium;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import com.deque.axe_core.commons.AxeWatcherOptions;
+import com.deque.axe_core.selenium.AxeWatcher;
+import com.deque.axe_core.selenium.AxeWatcherDriver;
+
+
+@DisplayName("My Login Application")
+class ManualModeTest {
+
+    WebDriver driver;
+    String apiKey = "10366b4a-80f2-4dcf-90b0-d9c226e27db7";
```

Comment:
Did you mean to do this? The other example uses `test_api_key` so this should probably do the same. Also hope this wasn't a real key.

```suggestion
    String apiKey = "test-api-key";
```